code/modules/mining/machine_vending.dm

```diff
@@ -300,5 +287,4 @@
 	new /obj/item/clothing/suit/hooded/explorer(src)
 	new /obj/item/encryptionkey/headset_cargo(src)
 	new /obj/item/clothing/mask/gas/explorer(src)
-	new /obj/item/card/mining_access_card(src)
-	new /obj/item/card/id/mining(src)
```

Comment:
undocumented change

Reply:
Technically, the functionality of the two items was merged and the card is thus no longer necessary.